Readme.md

```diff
@@ -103,10 +103,10 @@ introduction: 'Escrevendo CSS sem perder a sanidade. Com essa introdução, Rico
 
 In order to compile the assets and run Jekyll on local you need to follow those steps:
 
-- Install [NodeJS](https://nodejs.org/)
-- Run `npm install`
-- Run `npm install -g gulp gulp-cli` (maybe you need to use `sudo` command)
-- Run `gulp`
+- Install [NodeJS](https://nodejs.org/) (remember to use the latest version)
+- Run `sudo npm install`
+- Run `sudo npm install -g gulp gulp-cli`
+- Run `sudo gulp`
```

Comment:
Some people need to use `sudo` other don't. I don't know if we need to put this here...